torch/autograd/_functions/blas.py

```diff
@@ -21,17 +21,9 @@ def symbolic(g, add_matrix, matrix1, matrix2, beta=1, alpha=1, inplace=False):
         sizes1 = matrix1.type().sizes()
         sizes2 = matrix2.type().sizes()
         sizes_add = add_matrix.type().sizes()
-        broadcast = False
-        not_supported = False
         assert len(sizes1) == 2 and len(sizes2) == 2 and len(sizes_add) <= 2 and len(sizes_add) > 0
-        if len(sizes_add) == 1:
-            if sizes_add[0] == sizes2[1]:
-                broadcast = True
-            else:
-                not_supported = True
-        elif sizes1[0] != sizes_add[0] or sizes2[1] != sizes_add[1]:
-                not_supported = True
-        if not_supported:
+        broadcast, supported = check_onnx_broadcast([sizes1[0], sizes2[1]], sizes_add)
+        if not supported:
```

Comment:
Minor nit: the error message should probably go in the helper, since you'd probably want to raise it any time you use the broadcast checker. At the moment it doesn't matter because check_onnx_broadcast only has one call site.